src/app/app.routes.ts

```diff
@@ -7,6 +7,7 @@ import { AuthGuard } from '../user-profile/guards/auth.guard';
 import { AccessibilityComponent, CookiePolicyComponent, GetHelpComponent, PrivacyPolicyComponent, ServiceDownComponent, SignedOutComponent, TermsAndConditionsComponent } from './components';
 import { RedirectComponent } from './containers/redirect/redirect.component';
 import { TermsConditionGuard } from './guards/termsCondition.guard';
+import { TermsAndConditionsRegisterOtherOrgComponent } from './components/terms-and-conditions-register-other-org/terms-and-conditions-register-other-org.component';
```

Comment:
incorrect import ordering

Reply:
Thank you @connorpgpmcelroy I updated the code now.